.github/workflows/test.yaml

```diff
@@ -0,0 +1,41 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
```

Comment:
This part worries me.

Reply:
This document is provided by the following commit, since they changed organization from https://github.com/actions to https://github.com/ruby 
https://github.com/actions/starter-workflows/commit/67ffe1ea7ac8dc746ead8838b1a233a36d586bb3

When we proceed with a recommended workflow by GitHub, this comment is automatically written to the YAML.
![image](https://user-images.githubusercontent.com/4468155/99754397-a8c4f400-2b2b-11eb-912b-203a28841e23.png)

I think that ruby is a trusted organization, and this is not so troublesome.

Reply:
As those documents are thought to be provided as a context of liability limitation by GitHub, I will try making `ci.yaml` without the descriptions.

Reply:
I think ok to use actions from [https://github.com/actions](https://github.com/actions).
GitHub documentations suggested [https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-ruby](https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-ruby)  to use [https://github.com/actions](https://github.com/actions). Also actions packages provided by GitHub has MIT License https://github.com/actions/setup-ruby/blob/main/LICENSE